test/basic.test.ts

```diff
@@ -13,6 +13,7 @@ describe('tailwindcss module', async () => {
 
   await setupNuxtTailwind({
     exposeConfig: true,
+    exportLevel: 1,
```

Comment:
Shouldn't it be `exposeLevel`? And value of 2?

Reply:
Yes - my apologies! Missed that, but the default behaviour passed the test. Fixed now